README.md

```diff
@@ -24,6 +24,20 @@ jsonQuery.query("$..[?class]", data)
 
 [How to Use](https://dojotoolkit.org/reference-guide/1.10/dojox/json/query.html)
 
+# Additional features #
+JSON Query contains some enhancments not found in the original implementation.
+These include:
+1) Path in results (Currently not supported when running in browser):
+    It is often useful to also recieve the paths to the found items.
+    By setting "pathPropName" when creating the JSON Query object, a property name containing the path will be created in each sub-object of the data.
+    ** NOTE: This operation adds properties to the data
+
+    Example:
+    ```
+    var jsonQuery = require('query');
+    var _res = jsonQuery({pathPropName: "__path__"}).query("$..[?year>1975]", data)
+    ```
+
```

Comment:
Please add sample output for this code snippet.

Reply:
Done